server/mdm/linux/README.md

```diff
@@ -0,0 +1,36 @@
+# Linux encryption key sequence
+
+```mermaid
+sequenceDiagram   
+    participant MyDevicePage as My Device Page
```

Comment:
"My device" page on Fleet desktop

PS: To make clear the feature requires Fleet desktop.